addons/binding/org.openhab.binding.foobot/src/main/java/org/openhab/binding/foobot/FoobotBindingConstants.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.foobot;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link FoobotBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Divya Chauhan - Initial contribution
```

Comment:
@georgeFoobot make sure you are also properly credited on any applicable places and also make sure that the sign-off is correct, so look for `Also-by`

https://www.openhab.org/docs/developer/contributing/contributing.html